text/0113-canonical-metadata.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Canonical specification of metadata for LWC modules
+status: DRAFT
+created_at: 2020-05-07
+updated_at: 2020-05-07
+rfc: 
+champion: Aliaksandr Papko (@apapko) | Ravi Jayaramappa (@ravijayaramappa)
+implementation:
+---
+
+# Canonical specification of metadata for LWC modules
+
+Table of Content:
+* [Summary](#summary)
+* [Goals](#goals)
+* [Use Cases](#use-cases)
+* [Current State of Art](#current-state-of-art)
+* [Detailed design](#detailed-design)
+* [References](#reference)
+
+# Summary
+> The [dictionary definition](https://www.merriam-webster.com/dictionary/canonical%20form) of **Canonical form** is `the simplest form of something`. 
+
+This RFC aims to document the metadata collected by statically analyzing the source code of Lightning Web Component(LWC) modules. It documents the standardized shape of metadata.
```

Comment:
Are you aligned with @moses-salesforce on ensuring this is an exact subset of the broader type used by that workgroup to describe a component across any runtime?